### PR TITLE
sci-libs/opencascade-7.7.0 - package bump

### PR DESCRIPTION
**Pull request by waebbl**
**profile/base: mask tbb use flag for opencascade-7.7.0**

Building with TBB enabled is broken in the v7.7.0 release.

Bug: https://tracker.dev.opencascade.org/view.php?id=33152
Signed-off-by: Bernd Waibel <waebbl-gentoo@posteo.net>

**sci-libs/opencascade: add 7.7.0**

Bug: https://tracker.dev.opencascade.org/view.php?id=32650
Bug: https://tracker.dev.opencascade.org/view.php?id=33216
Closes: https://bugs.gentoo.org/880233
Signed-off-by: Bernd Waibel <waebbl-gentoo@posteo.net>

### CODE REVIEW

**Review by juippis — COMMENTED** [review]
Sorry but could you rebase? The package.use.mask file doesn't apply anylonger. I should react quickly now for any updates to this PR.

**Comment by waebbl** [comment]
@juippis rebased and pushed

File: profiles/base/package.use.mask [hunk 1]
```diff
@@ -6,6 +6,11 @@
 # This file is only for generic masks. For arch-specific masks (i.e.
 # mask everywhere, unmask on arch/*) use arch/base.
 
+# Bernd Waibel <waebbl-gentoo@posteo.net> (2022-11-25)
+# USE=tbb is broken in 7.7.0
+# https://tracker.dev.opencascade.org/view.php?id=33152
+=sci-libs/opencascade-7.7.0 tbb
```
Comment (juippis) [inline profiles/base/package.use.mask:12]: We should most likely use ~ here, not =.

**Review by juippis — APPROVED** [review]
Thanks!

File: profiles/base/package.use.mask (hunk 1 above)
Comment (waebbl) [reply profiles/base/package.use.mask:12]: yes you're right, it's likely we will have some revbumps before it get's fixed upstream

**Pull request closed without merging**